content/doc/index.adoc

```diff
@@ -4,26 +4,45 @@ title: Jenkins Documentation
 section: doc
 ---
 
+== What is Jenkins?
+
 Jenkins is a self-contained, open source automation server which can be used to
 automate all sorts of tasks such as building, testing, and deploying software.
+
 Jenkins can be installed through native system packages, Docker, or even run
-standalone by any machine with the Java Runtime Environment installed.
+standalone by any machine with the Java Runtime Environment (JRE) installed.
 
 == Guided Tour
 
-This guided tour will use the "standalone" Jenkins distribution which requires
-Java 8. A system with more than 512MB of RAM is also recommended.
+This guided tour introduces you to the basics of using Jenkins and its main
+feature, Jenkins Pipeline. This tour uses the \'standalone' Jenkins distribution,
+which runs locally on your own machine.
+
+=== Prerequisites
+
```

Comment:
We should note these are the prerequisites for this tour and that requirements for Jenkins depend on usage.

Reply:
I've now clarified that with an intro line under this heading.